src/Tracker/Survey.php

```diff
@@ -400,7 +400,7 @@ public function getQuestionInformation(string|null $language = null): array
      */
     public function getQuestionList(string|null $language = null): array
     {
-        return $this->getSource()->getQuestionList($language, $this->id, $this->getSourceSurveyId());
```

Comment:
Alle aanroepen van getQuestionList() die ik kan vinden hebben $language als eerste parameter. Volgens mij is dit niet juist.